packages/components/src/search/autocompleters/customers.js

```diff
@@ -13,6 +13,9 @@ import { stringifyQuery } from '@woocommerce/navigation';
  * Internal dependencies
  */
 import { computeSuggestionMatch } from './utils';
+import { SWAGGERNAMESPACE } from 'store/constants';
```

Comment:
I know you're still working on this, but this file won't be available outside the project- can you define `SWAGGERNAMESPACE` in this file instead? Or not use it, we ended up removing the NAMESPACE const in the other autocompleters.